source/Irrlicht/CNullDriver.cpp

```diff
@@ -328,17 +309,13 @@ void CNullDriver::removeFrameBuffer(IFrameBuffer* framebuf)
 
 void CNullDriver::removeAllMultisampleTextures()
 {
-	setMaterial ( SGPUMaterial() );
```

Comment:
this would be unbind all descriptor sets

Reply:
@Crisspl are we bothering with `removeAllTextures` and `removeAllMultisampleTextures` ?

Reply:
i dont know what u mean

Reply:
since textures and multisample textures are share-able there should be no reason for us to track them in the driver.

Reply:
resolved